src/neuroconv/datainterfaces/ecephys/openephys/openephys_utils.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from pathlib import Path
+
+from lxml import etree
+
+from neuroconv.utils import FolderPathType
+
+
+def get_session_start_time(element: etree.Element) -> datetime:
+    """
+    Get the session start time from the settings.xml file.
+
+    Parameters
+    ----------
+    element : etree.Element
+        The root element of the XML tree.
+    """
+    date_str = element.findtext("./INFO/DATE")
+    if date_str is None:
+        raise ValueError("Could not fetch session start time from settings.xml file.")
```

Comment:
maybe this could be a warning

Reply:
we can make this optional, changed in [9b796d9](https://github.com/catalystneuro/neuroconv/pull/971/commits/9b796d982acdbeca7c070a0298cd53916fd95a87)